setup.py

```diff
@@ -253,7 +252,11 @@ def run(self):
 extras["integrations"] = extras["optuna"] + extras["ray"]
 
 extras["serving"] = deps_list("pydantic", "uvicorn", "fastapi", "starlette")
-extras["speech"] = deps_list("soundfile", "torchaudio")
+extras["audio"] = deps_list("soundfile")
+extras["speech"] = deps_list("torchaudio") + extras["audio"]  # `pip install ".[speech]"` is deprecated and `pip install ".[torch-speech]"` should be used instead
```

Comment:
leaving a comment stating the `pip install ".[speech]"` is deprecated. Didn't manage to find a good way to print it out when running `pip install ".[speech]"` since the logs are bloated and there doesn't seem to be a clean way to do it....@Lysandre do you maybe have a trick here? 

https://stackoverflow.com/questions/44616823/how-to-print-warnings-and-errors-when-using-setuptools-pip

Reply:
No I don't think this can be done cleanly, I mostly meant not advertising the `speech` option in the notebooks/examples anymore